src/components/organisms/AdminTabsBar/AdminTabsBar.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useMemo } from "react";
+import { useParams } from "react-router";
+import { Link } from "react-router-dom";
+import { faClock, faPlayCircle } from "@fortawesome/free-regular-svg-icons";
+import {
+  faBorderNone,
+  faSlidersH,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { adminNGSettingsUrl, adminNGVenueUrl } from "utils/url";
+
+import "./AdminTabsBar.scss";
+
+export enum AdminTabsBarTab {
+  spaces = "spaces",
+  timing = "timing",
+  run = "run",
+  advanceSetting = "advance",
+}
+
+const adminVenueTabLabelMap: Readonly<Record<AdminTabsBarTab, String>> = {
+  [AdminTabsBarTab.spaces]: "Spaces",
+  [AdminTabsBarTab.timing]: "Timing",
+  [AdminTabsBarTab.run]: "Run",
+  [AdminTabsBarTab.advanceSetting]: "Advance setting",
+};
+
+export interface AdminNavBarRouteParams {
+  venueId?: string;
+  selectedTab?: AdminTabsBarTab;
+}
+
+const tabIcons: Record<string, IconDefinition> = {
+  [AdminTabsBarTab.spaces]: faBorderNone,
+  [AdminTabsBarTab.timing]: faClock,
+  [AdminTabsBarTab.run]: faPlayCircle,
+  [AdminTabsBarTab.advanceSetting]: faSlidersH,
+};
+
+export interface AdminTabsBarProps {
+  activeNav?: string;
+}
+
+export const AdminTabsBar: React.FC<AdminTabsBarProps> = ({
+  activeNav = "",
+  children,
+}) => {
+  const { venueId } = useParams<AdminNavBarRouteParams>();
+
+  const adminTabsUrl = useCallback(
+    (key: string) => {
+      return key === AdminTabsBarTab.advanceSetting
+        ? adminNGSettingsUrl(venueId)
```

Comment:
It should be using the same route, not another one.
If I can access the spaces like this:
https://co-reality-staging--preview-pr-1991-4osrrjuw.web.app/admin-ng/venue/8mars/spaces
Then I should be able to access advanced settings like this:
https://co-reality-staging--preview-pr-1991-4osrrjuw.web.app/admin-ng/venue/8mars/advanced

In the current implementation it's another route which is incorrect.

Ideally you should show this to Sofi and have a conversation about it. IMO only the "Advanced" Should be moved here and the other 2 tabs should exist in the other route.